quipucords/fingerprinter/runner.py

```diff
@@ -106,7 +101,7 @@ def format_certs(redhat_certs):
         """
         try:
             return [int(cert.strip(".pem")) for cert in redhat_certs if cert]
-        except Exception:  # pylint: disable=broad-except
+        except Exception:
             return []
```

Comment:
😢 

Again, not saying we need to fix things like this right now, but this one made me _oof_.